static/client.js

```
@@ -1013,10 +1020,31 @@ function getMedicationQuantity(row) {
     value: row["Qty"],
     unit: row["UoM"],
     system: "http://snomed.info/sct",
-    code: "385024007",
+    code: getMedicationQuantityCode(row["UoM"]),
   };
 }
 
+// todo: move this code to new column in test-pack or can we do snomed lookups?
+function getMedicationQuantityCode(unitsOfMeasure) {
+  switch (unitsOfMeasure) {
+    case "tablet":
+      return "428673006";
+    case "dose":
+      return "3317411000001100";
+    case "capsule":
+      return "428641000";
+    case "ampoule":
+      return "413516001";
+    case "vial":
+      return "415818006";
+    case "enema":
+      return "700476008";
+    case "device":
+    default:
+      return "999999999";
```

Do we want to fail here instead of sending an invalid code?

---

There are a lot of different UoMs on the test pack which I couldn't find snomed codes for so rather it "worked" than rejected till we found them all?

---

I'll see if I can find them in DM+D but yeah, fine to leave for now